src/support/file.h

```diff
@@ -28,16 +28,28 @@
 #include <vector>
 
 namespace wasm {
+
+namespace Flags {
+  enum BinaryOption {
+    Binary,
+    Text
+  };
+  enum DebugOption {
+    Debug,
+    Release
+  };
+}
```

Comment:
Then you'd need two such enums (to avoid mixing them up), and can't write `Flags::`, though, right? Currently you can write `Flags::` and it still avoids mixing them up, I believe.